### PR TITLE
Updating .pylintrc File and Clean Up Style Nits

### PR DESCRIPTION
**Pull request by xwsxethan**
This PR was from a looooooooooooooooong time ago, but I finally went back and found Landscape.io working again (thankfully). So I've updated the pylintrc file for our current style, except that I realized we still don't even follow that :| So I'll probably do a couple rounds of cleanup to get this to where we want it, but for now I've just done what I could. Feel free to fix anything you see as we do other PR's to make this smoother.

[Review on Reviewable](https://reviewable.io/reviews/uproxy/ufo-management-server-flask/28)

### CODE REVIEW

**Comment by jpevarnek** [comment]
Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[.pylintrc, line 13 \[r1\]](https://reviewable.io:443/reviews/uproxy/ufo-management-server-flask/28#-K9mh92lirjyIae7-tyy:-K9mh92lirjyIae7-tyz:1569116578) ([raw file](https://github.com/uproxy/ufo-management-server-flask/blob/b930329785d55879b4824f8b988229579f3a67e6/.pylintrc#L13)):*
I'm not sure if I totally understand pylint, why is this in multiple files?

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/ufo-management-server-flask/28)*

**Comment by xwsxethan** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[.pylintrc, line 13 \[r1\]](https://reviewable.io:443/reviews/uproxy/ufo-management-server-flask/28#-K9mh92lirjyIae7-tyy:-K9mhM5o7U_9mLY1fDl3:-1705468329) ([raw file](https://github.com/uproxy/ufo-management-server-flask/blob/b930329785d55879b4824f8b988229579f3a67e6/.pylintrc#L13)):*
I think landscape is acting up and not respecting this file, so this was my attempt to help that if it was somehow not applying the higher up .pylintrc file to sub directories. It appears that it is just not applying either file to any directories.

Also, sorry for any spam with this PR. This was mainly for testing landscape and figuring out when it will work, so I needed a PR to trigger their checks. This PR may be around for a while until it is fixed, but I'll ping you or Henry when we actually need to do anything.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/ufo-management-server-flask/28)*

**Comment by jpevarnek** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[.pylintrc, line 13 \[r1\]](https://reviewable.io:443/reviews/uproxy/ufo-management-server-flask/28#-K9mh92lirjyIae7-tyy:-KA1MqqTpwMPBctV1Wy2:-545808432) ([raw file](https://github.com/uproxy/ufo-management-server-flask/blob/b930329785d55879b4824f8b988229579f3a67e6/.pylintrc#L13)):*
Okay, it would be nice if we could figure out some explanation for why that's happening before this becomes a real PR.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/ufo-management-server-flask/28)*

**Comment by xwsxethan** [comment]
Reviewed 19 of 20 files at r2, 7 of 7 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/uproxy/ufo-management-server-flask/28)*

File: clock.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 SCHEDULER = BlockingScheduler()
 
 @SCHEDULER.scheduled_job('interval', minutes=15)
-def distribute_user_keys_to_proxy_servers():
+def schedule_user_key_distribution():
```
Comment (blueandgold) [inline clock.py]: Also change "sync_users()" below to be consistent with this change?
```python
schedule_user_sync()
```

**Comment by blueandgold** [comment]
:+1:  Very cool!

**Comment by xwsxethan** [comment]
Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[clock.py, line 21 \[r3\]](https://reviewable.io:443/reviews/uproxy/ufo-management-server-flask/28#-KKok-YI6y8O-uzeseeH-r3-21:-KKsShmw5zkPU0nPTCY0:-499146009) ([raw file](https://github.com/uproxy/ufo-management-server-flask/blob/01f44889cdfeec4fe5f7c0d72b7b163e91420401/clock.py#L21)):*
<details><summary><i>Previously, blueandgold (Henry Chang) wrote…</i></summary>
> Also change "sync_users()" below to be consistent with this change?
> 
> ```python
> 
> schedule_user_sync()
> 
> ```
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/uproxy/ufo-management-server-flask/28)*

**Pull request merged**